Package/nuget/DynamicLanguageRuntime.nuspec

```diff
@@ -24,6 +24,12 @@
         <dependency id="System.CodeDom" version="4.5.0" />
         <dependency id="System.Configuration.ConfigurationManager" version="4.5.0" />
       </group>
+      <group targetFramework="netstandard">
+        <dependency id="System.CodeDom" version="4.5.0" />
+        <dependency id="System.Configuration.ConfigurationManager" version="4.5.0" />
+        <dependency id="System.Reflection.Emit" version="4.6.0-preview.19073.11" />
```

Comment:
I am using the prerelease versions as I believe (but am not sure) that the prerelease versions will throw a PlatformNotSupportedException on AOT platforms, whereas the stable versions wont. I hope to give that a test at some point.

Reply:
Tested, and I am correct. Stable will throw this exception:

> System.Exception: 'Method 'TypeBuilder.DefineConstructor(MethodAttributes, CallingConventions, Type[])' was not included in compilation, but was referenced in DelegateUtils.EmitCallSiteDelegateType(int). There may have been a missing assembly.'

The preview will throw this, more helpful exception:

> System.PlatformNotSupportedException: PlatformNotSupported_RefEmit. For more information, visit http://go.microsoft.com/fwlink/?LinkId=623485